code/drasil-example/Drasil/SWHS/IMods.hs

```diff
@@ -204,7 +205,7 @@ eBalanceOnPCM = im eBalanceOnPCM_rc [qw temp_melt_P, qw time_final, qw temp_init
  qw pcm_HTC, qw pcm_mass, qw htCap_S_P, qw htCap_L_P]
   [sy temp_init $< sy temp_melt_P] (qw temp_PCM)
    [0 $<= sy time $<= sy time_final] [makeCite koothoor2013] eBalanceOnPCMDeriv 
-   "eBalanceOnPCM" [balPCMDesc_note]
+   "eBalanceOnPCM" balPCMDescNotes
 
 eBalanceOnPCM_rc :: RelationConcept
 eBalanceOnPCM_rc = makeRC "eBalanceOnPCM_rc" (nounPhraseSP
```

Comment:
In this `RelationConcept`, it takes in a completely different `Sentence` from the Notes - should `RelationConcept`s be taking these `Sentence`s? @JacquesCarette